explainers/hidden-content-explainer.md

```diff
@@ -12,28 +12,21 @@ This is an explainer for two closely related features:
         hidden from the user (similar to `content-visibility: hidden`), but the
         content remains searchable via user-agent find-in-page algorithms.
 
-2. The `beforematch` event allows developers to display content to the user in
-    response to actions which scroll the page to a target element. It is fired
-    at render timing just before these actions scroll the page. The applicable
-    actions are the following:
-    * There is a new find-in-page (ctrl+f) active match, in which case the event
-      is fired with the matched element as the target.
-    * There is an element fragment navigation (`example.com/#foo`) or the fragment
-      is changed by script (`window.location.hash = ...`), in which case the event
-      is fired with the matched element as the target.
+2. The `beforematch` event allows developers to display `hidden-matchable`
+    content to the user in response to searches which scroll the page to some
+    target text. The event is fired at render timing on the nearest ancestor
+    block-level element for these cases:
+    * There is a new find-in-page (ctrl+f)
```

Comment:
You don't really need "ctrl+f" especially since it's "cmd+f" on mac :) I think find-in-page is unambiguous enough

Reply:
done